### ISSUE TITLE
Ability to stop comparison when threshold is met?

### ISSUE THREAD

**Issue by mthelm85** [matthieugomez/StringDistances.jl#14]
Hi Matthieu, 

Thanks for the awesome package! I was wondering if it would be possible to implement the ability to stop the comparison when a threshold is met? For example, if I'm only interested in filtering by strings where the result of ```compare``` is > 0.8, I'm wondering if it would be possible to know earlier whether or not two strings are going to exceed the threshold, and then bail out on the comparison? 

See [this thread on discourse for background](https://discourse.julialang.org/t/whats-the-best-way-to-compare-items-in-an-array-to-every-other-item/26806/11). 

Basically, I'm doing comparisons on arrays that contain hundreds of thousands of strings, so I'm looking for ways to optimize the comparison so that it doesn't take so long.

**Comment by matthieugomez** [comment]
If you use the lastest master, I have added a keyword `min_dist` to `compare` for `Levenshtein()` and `DamerauLevenshtein()`.
This is an experimental feature --- let me know if you find it useful.

**Comment by matthieugomez** [comment]
Going forward, the right way to do what you want to do in the thread is to use `find_best` or `find_all`
